### PR TITLE
browser(webkit): roll to 06/17/21

### PR DESCRIPTION
**Pull request by dpino**
This pull-request rebases `webkit/bootstrap.diff` to webkit ToT (a60349c98390c46d865cbaa55431b074c2144639).

It was necessary to disable Cog compilation as it was producing a compilation error in WPE.

### CODE REVIEW

**Comment by dpino** [comment]
Commit with resolution of conflicts after rebasing `webkit/bootstrap.patch`:

https://github.com/dpino/WebKit/commit/99fccd66c6172f4c96430974fdc9a86eee02e61d

File: browser_patches/webkit/BUILD_NUMBER [hunk 1]
```diff
@@ -1,2 +1,4 @@
 1499
 Changed: yurys@chromium.org Thu, Jun 10, 2021 11:01:06 PM
+1500
```
Comment (yury-s) [inline browser_patches/webkit/BUILD_NUMBER:3]: This entire file should have been overwritten and contain just one entry. Did you use `./browser_patches/export.sh webkit` to prepare the PR?

File: browser_patches/webkit/patches/bootstrap.diff [hunk 2]
```diff
@@ -3308,7 +3308,7 @@ index f52e4abc8e76c02285b9c68ac9c7a98bd203e0dd..a3f08d8c0644ac2ca61c66c979d3ae7a
 +            return makeUnexpected("Unable to decode given content"_s);
 +
 +        ScriptExecutionContext* context = element->scriptExecutionContext();
-+        fileObjects.append(File::create(context, Blob::create(context, SharedBuffer::create(WTFMove(*buffer)), type), name));
++	fileObjects.append(File::create(context, Blob::create(context, Vector { buffer->data(), buffer->size() }, type), name));
```
Comment (yury-s) [inline browser_patches/webkit/patches/bootstrap.diff:3311]: Just `Blob::create(context, WTFMove(*buffer))` to avoid extra copy?

**Review by yury-s — APPROVED** [review]

**Review by wezgreen — APPROVED** [review]

File: browser_patches/webkit/BUILD_NUMBER (hunk 1 above)
Comment (yury-s) [reply browser_patches/webkit/BUILD_NUMBER:1]: Edited it in place.

File: browser_patches/webkit/patches/bootstrap.diff (hunk 2 above)
Comment (yury-s) [reply browser_patches/webkit/patches/bootstrap.diff:3311]: Let's address it in follow-up, I'm going to land this PR and see if it builds on all platforms.

File: browser_patches/webkit/BUILD_NUMBER (hunk 1 above)
Comment (dpino) [reply browser_patches/webkit/BUILD_NUMBER:1]: I updated the file manually, but I thought I had change the committer name and date too. Next time I will use `./browser_patches/export.sh webkit`.

**Pull request merged**